[2024-12-03T16:01:22.368+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_file_processing.etl_process_fr manual__2024-12-03T16:01:12.895897+00:00 [queued]>
[2024-12-03T16:01:22.395+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_file_processing.etl_process_fr manual__2024-12-03T16:01:12.895897+00:00 [queued]>
[2024-12-03T16:01:22.398+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-12-03T16:01:22.430+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): etl_process_fr> on 2024-12-03 16:01:12.895897+00:00
[2024-12-03T16:01:22.475+0000] {standard_task_runner.py:57} INFO - Started process 105 to run task
[2024-12-03T16:01:22.498+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_file_processing', 'etl_process_fr', 'manual__2024-12-03T16:01:12.895897+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/ETL_file_processing.py', '--cfg-path', '/tmp/tmp3gnmizps']
[2024-12-03T16:01:22.504+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask etl_process_fr
[2024-12-03T16:01:22.795+0000] {task_command.py:410} INFO - Running <TaskInstance: etl_file_processing.etl_process_fr manual__2024-12-03T16:01:12.895897+00:00 [running]> on host effe9442f8f2
[2024-12-03T16:01:23.000+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_file_processing' AIRFLOW_CTX_TASK_ID='etl_process_fr' AIRFLOW_CTX_EXECUTION_DATE='2024-12-03T16:01:12.895897+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-03T16:01:12.895897+00:00'
[2024-12-03T16:01:28.493+0000] {logging_mixin.py:150} INFO - No data found in the 'frmodels' collection.
[2024-12-03T16:01:28.500+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-12-03T16:01:28.536+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=etl_file_processing, task_id=etl_process_fr, execution_date=20241203T160112, start_date=20241203T160122, end_date=20241203T160128
[2024-12-03T16:01:28.573+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-12-03T16:01:28.643+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
